arbitrum-docs/partials/_troubleshooting-building-partial.md

```diff
@@ -5,7 +5,7 @@
 
 <p>The L1 component is meant to compensate the Sequencer for the cost of posting transactions on L1 (but no more). (See <a href="https://developer.arbitrum.io/arbos/l1-pricing">L1 Pricing</a>.)</p>
 
-<p>The L2 component covers the cost of operating the L2 chain; it uses Geth for gas calculation and thus behaves nearly identically to L1 Ethereum. One difference is that unlike on Ethereum, Arbitrum chains enforce a gas price floor; currently 0.1 gwei on Arbitrum One and 0.01 gwei on Nova (See [Gas](/arbos/gas.mdx)).</p>
```

Comment:
non-blocking nit - should we normalize relative links over absolute links wherever possible?

Reply:
Agree. Links in Notion sourced content are more complicated to deal with, because they don't render if they are not actual links in Notion, so the relative links don't work. I'm already planning a revamp of the library to render links as relative (wherever possible), and render actual markdown instead of HTML here. I did a few tests and it shouldn't be too difficult to do.